packages/core/src/middleware/koa-pagination.ts

```diff
@@ -67,4 +78,6 @@ export default function koaPagination<StateT, ContextT, ResponseBodyT>({
       ctx.append('Link', buildLink(ctx.request, page + 1, 'next'));
     }
   };
+
+  return paginationMiddleware;
```

Comment:
What is this variable use for?

Reply:
i think it's for function name

Reply:
i did not expect this (i was thinking arrow function has no "_name"). maybe we can use `instanceof`?

Reply:
It's used to identify the pagination middleware.

Reply:
Added a comment to note it

```ts
  // Name this anonymous function for the utility function `isPaginationMiddleware` to identify it
  const paginationMiddleware: MiddlewareType<
    StateT,
    WithPaginationContext<ContextT>,
    ResponseBodyT
  > = async (ctx, next) => {

```